src/js/storyline.js

```diff
@@ -47,7 +56,20 @@ Storyline.prototype = {
     if (!this.dataConfig.chart.datetime_format) { this.dataConfig.chart.datetime_format = this.dataConfig.data.datetime_format }
 
   },
-  resetWidth: function(newWidth) {
+  initTracking: function() {
+    try {
+      var visitor = ua('UA-27829802-5',  {https: true});
+      visitor.pageview({dl: document.location.href}).send();
+      this.visitor = visitor;
+    } catch(e) { /* we don't want any problem here to sidetrack things */}
+  },
+  trackEvent: function( category, action, label, value ) {
+    if (this.visitor) {
+      this.visitor.event(category, action, label, value).send();
+    }
+  },
+  resetWidth: function(newWidth, targetType) {
+    storyline.trackEvent('resize', targetType)
```

Comment:
we should eliminate all references to a global `storyline` in favor of `this` (or `this.storyline` for components)